libs/api/utils/src/lib/api-utils.ts

```diff
@@ -1,8 +1,31 @@
+import { prisma } from 'apps/coursition/prisma/prismaClient'
```

Comment:
**suggestion:** (blocking) libs shouldn't depend on apps. The flow is well explained here:
https://dev.to/this-is-learning/architects-delight-enforcing-layers-and-project-boundaries-with-nx-2d8o

We should adopt this structure in Nx, let's have a discussion on it soon. :)
https://dev.to/this-is-learning/semantic-grouping-folders-with-nx-3467